lib/gym/project.rb

```diff
@@ -83,6 +83,7 @@ def build_settings(key: nil, optional: true)
         command = "xcrun xcodebuild -showBuildSettings #{BuildCommandGenerator.project_path_array.join(' ')}"
         Helper.log.info command.yellow unless Gym.config[:silent]
         @build_settings = `#{command}`
+        Helper.log.info @build_settings if $verbose
```

Comment:
> That's too much information, did you take a look at this?  

Yes. I personally don't mind it in verbose mode. Especially if it helps troubleshooting.(as --verbose is recommended when encountering an issue). It could help developers to fix issue faster.

You could replace it with something like:

````
Helper.log.info "If you have an issue, please consider running the above command and sending us its output as part of your report".green
`````

But I would prefer something that helps me diagnose variation between builds easily and this was my solution :)